src/Apps/Artwork/Components/OtherWorks/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button, Serif } from "@artsy/palette"
+import React from "react"
+
+interface HeaderProps {
+  title: string
+  buttonHref: string
+}
+
+export const Header: React.SFC<HeaderProps> = props => {
+  return (
+    <>
+      <Serif size="8" color="black100" mb={2}>
+        {props.title}
+      </Serif>
+      <Button
+        variant="secondaryOutline"
+        mb={3}
+        onClick={() => (window.location.href = props.buttonHref)}
```

Comment:
#minor but is it good to use a button (just a `div`), with JS needing to navigate? My first thought was this should be an `a` tag with an `href`, but just styled. That way it'll work even without JS (and isn't it more semantic to be an `a` tag since it is a link)?